css/index.css

```diff
@@ -323,17 +323,22 @@ header nav a {
 }
 .home .content-pick .destination .btn {
   width: 200px;
-  background-color: #17A2B8;
   padding: 14px 0px;
   text-align: center;
-  color: white;
   font-size: 2rem;
   font-family: 'Roboto', cursive;
   border-radius: 16px;
+  background-color: #17A2B8;
+  color: #FFFFFF;
 }
 @media (max-width:500px) {
   .home .content-pick .destination .btn {
     padding: 3% 5%;
     justify-content: center;
   }
 }
+@media (max-width:500px) {
+  .home .content-pick .destination .btn {
+    width: 100%;
+  }
```

Comment:
If you do this:

```
@media (max-width:500px) {
    width: 100%;
}
```

And throw it inside the `.home .content-pick .destination .btn` selector, do you get the same result? This goes back to "nested media queries" we talked about in Preprocessing-I, but I'll it to you do the science and experiment!